nodejs/templates/dns.yaml

```diff
@@ -0,0 +1,46 @@
+{{ if .Values.registerAdditionalDns.enabled }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "hmcts.releaseName" . }}-dns
+  labels:
+    app.kubernetes.io/name: {{ template "hmcts.releaseName" . }}-dns
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ template "hmcts.releaseName" . }}
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-weight: "-5"
+    helm.sh/hook-delete-policy: before-hook-creation
+spec:
+  template:
+    metadata:
+      name: {{ template "hmcts.releaseName" . }}-dns
+      labels:
+        app.kubernetes.io/name: {{ template "hmcts.releaseName" . }}-dns
+    spec:
+      volumes:
+        - name: container-init
+          configMap:
+            name: {{ template "hmcts.releaseName" . }}-bin
+            defaultMode: 0755
+      containers:
+        - name: {{ template "hmcts.releaseName" . }}-dns
+          image: hmcts/curl
+          volumeMounts:
+            - name: container-init
+              mountPath: /container.init
+          command: [ "/bin/sh" ]
+          args:
+            - /container.init/nodejs-dns
+            - {{ .Values.registerAdditionalDns.prefix }}-{{ .Release.Name }}
```

Comment:
Shouldn't there be a check for this to be less than 63 characters (DNS spec)? 
Something like (I think): {{- printf "%s-%s" .Values.registerAdditionalDns.prefix  .Release.Name  | trunc 63 | trimSuffix "-" -}}

Reply:
this is just an arg though - doesn't it just create an entry with consul (not a k8s resource - where the trunc would be needed)?

Reply:
It's a DNS limit, which k8s uses for its resources as well. Probably not an issue in our case. Something to keep in mind though